app/helpers/repository_helper.rb

```diff
@@ -23,8 +23,7 @@ def month_options
   end
 
   def year_options
-    # FIXME: this will not work some years from now
-    (2013..2020).to_a.map {|year| [year, year]}
+    (DateTime.now.year..DateTime.now.year + 5).to_a.map {|year| [year, year]}
```

Comment:
https://github.com/mezuro/prezento/pull/370/files#r76909306 brought to my attention that this will not allow users to search the historic before the current year, which exists. And I hope we don't have processings coming from the future hahaha.. The previous code made that and now I see how weird it is. I'm sorry it led you to this...

What do you think about something like the following? `(DateTime.now.year - 100..DateTime.now.year)`

This should also mke your change to `repository/show/date_select.feature
` unnecessary.